src/osd/PrimaryLogPG.cc

```diff
@@ -3445,7 +3445,18 @@ int PrimaryLogPG::get_manifest_ref_count(ObjectContextRef obc, std::string& fp_o
 
 bool PrimaryLogPG::recover_adjacent_clones(ObjectContextRef obc, OpRequestRef op)
 {
-  if (!obc->obs.oi.manifest.is_chunked() || !obc->ssc || !obc->ssc->snapset.clones.size()) {
+  bool has_manifest_op = false;
```

Comment:
might want to move the definition of `has_manifest_op` closer to where it is set.